### DISCUSSION TITLE
Student application

### DISCUSSION THREAD

**Discussion by Joelthomas199** [community/community#126848] [GitHub Education] [answered]
### Hi everyone,

I got rejected for my application and have tried again after 3 days but it does not let me past the upload a id document screen any ideas what to do?
![Screenshot 2024-06-01 153140](https://github.com/community/community/assets/171253090/b8a5a931-573c-4655-856c-636f6c6dbb35)

**Comment by davevad93** [comment]
Hi @Joelthomas199 ,

Recently I saw another user who had the same problem as you, he solved it by uploading a front/back picture of his ID, to do that take the photo and then combine the two pictures into a single `JPEG` or `PNG` file. There was also another person who couldn't get past the upload screen and eventually was able to figure it out, here's the discussion #126276.

**Reply by JJDC982** [reply]
Beste Thimo,
Als je nog geen AI stageproject hebt gekozen neem even contact op met mij => jeanjacques.declercq@ugent.be
Mvg,   Prof. DE CLERCQ (Fac ECO, vakgroep "Research Cluster Big Data Analytics)
